app/views/notification_mailer/overdue_notification.text.erb

```diff
@@ -1,9 +1,16 @@
 <%= raw @info_request.public_body.name %> <%= _('have delayed.') %>
 
+<% if @info_request.public_body.not_subject_to_law? %>
+<%= _('They have not replied to your {{law_used_short}} request {{title}} ' \
```

Comment:
I think we can remove the conditionals on this and use "normally required by law" for both situations.